inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/mvn.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/mvn.hpp"
+
+#include <vector>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+    INSTANTIATE_TEST_CASE_P(smoke_MVN_4D, Mvn6LayerTest,
+                            ::testing::Combine(
+                                    ::testing::Values(std::vector<size_t>{80, 5, 7, 1}),
```

Comment:
Should the tests cover more cases of MVN layer?

Reply:
deprecated dldt tests with old MVN work fine?

Reply:
Yes, sure.

Reply:
I added more test cases and disabled deprecated tests while investigation is go on.